### PR TITLE
docs/typography

### PR DESCRIPTION
**Pull request by silvenon**
Storybook: https://orbit-docs-typography.surge.sh

### CODE REVIEW

File: docs/src/components/BaseStyles/BaseStyles.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { createGlobalStyle, css } from "styled-components";
+
+const BaseStyles = createGlobalStyle`
+  ${({ theme }) => css`
+    html {
+      font-family: ${theme.orbit.fontFamily};
+      font-size: 14px;
+    }
+
+    ${mediaQueries.tablet(css`
```
Comment (mainframev) [inline docs/src/components/BaseStyles/BaseStyles.tsx:11]: great, that finally `rems` make sense :)

File: docs/src/theme.tsx [hunk 2]
```diff
@@ -1,7 +1,27 @@
 import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
 export default {
-  orbit: { ...defaultTokens, widthBreakpointDesktop: 1024 },
+  orbit: {
+    ...defaultTokens,
+    widthBreakpointDesktop: 1024,
+    fontFamily: '"DM Sans", sans-serif',
+    fontSizeHeadingDisplay: "48px",
```
[the remark this answers is not in the document]
Comment (silvenon) [reply docs/src/theme.tsx:8]: I'm not sure whether I want that. For me it's only important that the body text is smaller. We'll see what Will says.

[the remark this answers is not in the document]
Comment (silvenon) [reply docs/src/theme.tsx:8]: Will said that even on mobile base font size should be 16px, so headings should stay the same as well.

File: docs/src/components/Head/index.tsx [hunk 3]
```diff
@@ -29,6 +29,11 @@ export default function Head({ title, hasSiteName, description, path }: Props) {
       <meta name="twitter:site" content="@OrbitKiwi" />
       <meta name="twitter:title" content={pageTitle} />
       <meta name="twitter:description" content={description} />
+      <link rel="preconnect" href="https://fonts.gstatic.com" />
+      <link
+        href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,500;0,700;1,400&display=swap"
```
Comment (mainframev) [inline docs/src/components/Head/index.tsx:34]: we use only English, so i think we can have a parameter `lang=en`

**Review by mainframev — APPROVED** [review]

**Pull request merged**